projects/plugins/jetpack/_inc/lib/class-jetpack-top-posts-helper.php

```diff
@@ -94,7 +94,7 @@ public static function get_top_posts( $period, $items_count, $types ) {
 					'context'   => get_the_category( $post_id ) ? get_the_category( $post_id ) : get_the_tags( $post_id ),
 					'href'      => $post['href'],
 					'date'      => get_the_date( '', $post_id ),
-					'title'     => $post['title'],
+					'title'     => strlen( $post['title'] ) > 90 ? sprintf( '%s...', substr( $post['title'], 0, 90 ) ) : $post['title'],
```

Comment:
Any preference on what an appropriate character limit might be? 90 felt reasonable, but happy to change this.

Reply:
Actually, I do wonder if we should limit characters here. I agree that long titles don't look great in a column of three, but they're a lot more acceptable in the List View, and it's probably harder to restrict a limit to only one view without rendering two titles. 

It might just be that while long titles are inadvisable and might not look great, that should be up to the user rather than imposing an arbitrary character count? The wrapping should improve the display anyway.

Any thoughts?!